ci/arch-linux/PKGBUILD

```
@@ -0,0 +1,82 @@
+# Maintainer: Hiroshi Hatake <cosmo0920.wp[at]gmail.com>
+
+pkgname=groonga
+pkgdesc="An open-source fulltext search engine and column store."
+pkgver=14.0.4
```

Should we update the package version here?
```suggestion
pkgver=15.0.9
```

---

https://github.com/kou/groonga/blob/c68050262e0c96c1172134290efa479e10612710/ci/arch-linux/build.sh#L26-L28

ここの処理でバージョンを最新版に`sed`で変えてるから実はその部分はテンプレ的なもので
実際はどのバージョンでも良いのかも？（認識違ってたらごめんなさい）

---

たしかにです！そうすると空になっていると、後で置換する感があって良さそうですかね。
```suggestion
pkgver=
```
ref: https://github.com/groonga/groonga/blob/4eee0caeef3bacf718b7c90e97a20b90f707f01a/ci/msys2/PKGBUILD#L6

---

空よりかは`pkgver=TEMPLATE_VER_WILL_BE_REPLACED`とかプレースホルダー的なものがあった方が個人的には好きです。

元々の`14.0.4`でも良いかもですが。

---

こいつがマージされたらリリースプロセスの一環でこいつとチェックサムを自動更新してこの`PKGBUILD`をコピーしてAURにpushするまでを自動化するつもりです。なので、こいつは最終的には常に最新のバージョンを指すようになります。

---

テンプレ的なファイルに決めうちのバージョンが書いてあると、後からこのファイルだけ見た人が常にバージョンは14.0.4なの？と勘違いしてしまう可能性を排除したい的な意味ですかね？

---

> こいつがマージされたらリリースプロセスの一環でこいつとチェックサムを自動更新してこのPKGBUILDをコピーしてAURにpushするまでを自動化するつもりです。なので、こいつは最終的には常に最新のバージョンを指すようになります。

後続で対応するため、現状は気にする必要がないことを理解しました！